index.js

```diff
@@ -33,7 +34,8 @@ function handleArgs(cmd, args, opts) {
 		stripEof: true,
 		preferLocal: true,
 		encoding: 'utf8',
-		reject: true
+		reject: true,
+		cleanup: false
```

Comment:
Any reason we wouldn't want this as `true` by default? I already assumed that's how the core one worked. We could clearly document it in the readme.